veg2hab/io/common.py

```diff
@@ -135,14 +146,20 @@ class Config:
         ),
         description="Minimum oppervlakken per habitattype",
     )
+
+    @property
```

Comment:
Je kunt dit ook nog doen tijdens het inlezen. Volgens mij moet dat dan via een validator. Dus dan is minimum_oppervlak_exceptions gewoon de dict, maar dan pas je dus eigenlijk de parsing aan dat als ie een string krijgt, deze eerst ge "json load wordt"

Reply:
Goed excuus om validators eens uit te pluizen :)